ThreadGroup/Controllers/TGMainViewController.m

```diff
@@ -199,9 +203,13 @@ - (void)configureMainViewForViewState:(TGMainViewState)viewState {
             break;
         case TGMainViewStateConnectDevicePassphrase:
         case TGMainViewStateConnectDeviceScanning:
-            [self resetSelectDeviceView];
-            [self setPopupNotificationForState:viewState animated:YES];
-            break;
+            if ([self isTogglingBetweenScanningAndPassphraseViewStates]) {
```

Comment:
We don't need to worry about this anymore. We should always return the user to TGMainViewStateLookingForRouter when we configureUIForReachableState